main/test/src/main/ClientServerTests.scala

```diff
@@ -42,6 +42,12 @@ class EchoServer extends MillServerMain[Int] {
 
 object ClientServerTests extends TestSuite {
 
+  if (scala.util.Properties.isWin) {
+    // workaround for CI issue
+    // https://github.com/com-lihaoyi/mill/pull/1852#issuecomment-1114332274
+    System.setProperty("os.name", "Windows10")
```

Comment:
Seems legit, although I'd like to find a solution, which does not tamper with JVM default properties. Maybe, setting the `org.newsclub.net.unix.library.override` instead also works around the wrong detection on Windows?

Reply:
Might not work that easily, as this property is meant to point to an unpacked library.

I will merge as is. If you by any change find a better solution, please open a new PR.

Reply:
Yeah, thought about it but it requires a full path. I will make an issue or fix upstream.